lib/interfaces/common.js

```diff
@@ -147,7 +149,7 @@ module.exports = function(suites, context, mocha) {
           }
           suites.shift();
         } else if (typeof opts.fn === 'undefined' && !suite.pending) {
-          throw new Error(
+          throw MissingCallbackError(
```

Comment:
seems hinky **not** to be using `new` when _creating_ error objects...

Reply:
@plroebuck I will update the names (+ make camelCased), but plan to leave as factories without `new`. Let me know if you feel strongly about using the other way round.

Reply:
Well, using `create<xxx>Error` _somewhat_ implies `new`, so naming-wise okay.
**Very** disappointed we couldn't use _actual_ Error objects instead since now the stacks will be a level deeper than the actual problem...